docs/content/en/docs/pipeline-stages/builders/custom.md

```diff
@@ -9,17 +9,19 @@ Custom build scripts allow skaffold users the flexibility to build artifacts wit
 Users can write a custom build script which must abide by the following contract for skaffold to work as expected:
 
 Currently, this only works with [local](#custom-build-script-locally) and 
-[cluster](#custom-build-script-in-cluster) build types. 
+[cluster](#custom-build-script-in-cluster) build types.
+
 ### Contract between Skaffold and Custom Build Script
 
 Skaffold will pass in the following environment variables to the custom build script:
 
 | Environment Variable         | Description           | Expectation  |
 | ------------- |-------------| -----|
+| Local environment variables | The current state of the local environment (e.g. `$HOST`, `$PATH)`. Determined by the golang [os.Environ](https://golang.org/pkg/os#Environ) function.| |
+| User-defined environment variables | Environment variables specified in a `key=value` form, in the `env` section of the `custom` configuration. | |
 | $IMAGE     | The fully qualified image name. For example, "gcr.io/image1:tag" | The custom build script is expected to build this image and tag it with the name provided in $IMAGE. The image should also be pushed if `$PUSH_IMAGE=true`. | 
 | $PUSH_IMAGE      | Set to true if the image in `$IMAGE` is expected to exist in a remote registry. Set to false if the image is expected to exist locally.      |   The custom build script will push the image `$IMAGE` if `$PUSH_IMAGE=true` | 
-| $BUILD_CONTEXT  | An absolute path to the directory this artifact is meant to be built from. Specified by artifact `context` in the skaffold.yaml.      | None. | 
-| Local environment variables | The current state of the local environment (e.g. `$HOST`, `$PATH)`. Determined by the golang [os.Environ](https://golang.org/pkg/os#Environ) function.| None. |
+| $BUILD_CONTEXT  | An absolute path to the directory this artifact is meant to be built from. Specified by artifact `context` in the skaffold.yaml.      | |
```

Comment:
This change in order seems to indicate the precedence?  Might be worth explicitly calling that out?

Reply:
I've re-focused this PR on the buildpacks builder only